staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1105,6 +1105,9 @@ type ManagedFieldsEntry struct {
 	// Time is timestamp of when these fields were set. It should always be empty if Operation is 'Apply'
 	// +optional
 	Time *Time `json:"time,omitempty" protobuf:"bytes,4,opt,name=time"`
+	// FieldsType identifies the format of raw bytes in Fields below.
+	// There is currently only one possible value: "json+v1"
```

Comment:
I think you also need to document what happens if the object is json and the Fields are some binary format?